src/data/DocenteDatabase.ts

```diff
@@ -0,0 +1,85 @@
+import dotenv from 'dotenv'
+import { v4 as generateId } from 'uuid'
+import { BaseDatabase } from './BaseDatabase'
+import { Docente } from './Docente'
+
+dotenv.config()
+
+export class DocenteDataBase extends BaseDatabase {
+
+    public getDocentes = async () => {
+        try {
+            const result = await this.connection('LabenuSystem_Docente')
+            .select('*')
+
+            return result
+        }
```

Comment:
Se, depois do await, não se faz nada com o resultado, não é necessário usar async/await. Poderia fazer apenas return this.connection(...). O código ficaria mais curto, e retornaria uma Promise do mesmo jeito.